### PR TITLE
Update list of shipping locales on iOS + add ar as experiment

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

**Comment by TheoChevalier** [comment]
Using another approach than adding ar to App Store supported locales.

Also removing $android_locales_marketing as it seems unused.

**Comment by pascalchevrel** [comment]
The view `locale/ar/google/release/ ` and `/locale/ar/google/release/html/ ` should have the \<pre\> fields displayed as RTL.

**Comment by TheoChevalier** [comment]
updated

File: app/classes/Stores/Project.php [hunk 1]
```diff
@@ -197,6 +195,14 @@ public function __construct()
     }
 
     /**
+     * Return true if $locale is a Right-To-Left locale, false otherwise.
+     */
```
Comment (pascalchevrel) [inline app/classes/Stores/Project.php]: incomplete docbock :)

File: app/controllers/locale_controller.php [hunk 2]
```diff
@@ -35,6 +35,7 @@
 }
 
 $title = ucfirst($request['store']) . " Store Description, {$request['channel']} channel, for: {$request['locale']}";
+$direction = $project->isRTL($request['locale']) ? 'dir="rtl"' : 'dir="ltr"';
```
Comment (pascalchevrel) [inline app/controllers/locale_controller.php]: I think it would make more sense to put that line in the model

**Pull request merged**